<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DungeonGenerateDialog</class>
 <widget class="QDialog">
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="levelLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Difficulty:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="difficultyLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Multi:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="multiCheckBox">
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Check if it is for a multiplayer game</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Seed:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEditWidget" name="seedLineEdit">
        <property name="charWidth">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Quest:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="LineEditWidget" name="questSeedLineEdit">
        <property name="charWidth">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
